ansible/roles/concord-demo/templates/concord.conf.j2

```diff
@@ -0,0 +1,16 @@
+description "concord"
+
+start on startup
+
+respawn
+
+env LD_LIBRARY_PATH=/usr/local/lib
+env MESOS_NATIVE_JAVA_LIBRARY=/usr/local/lib/libmesos-{{ concord_mesos_version }}.so
+exec java {% for k, v in concord_java_options.iteritems() %}-{{k}}={{v}} {% endfor %} -cp \
```

Comment:
what's w/ the -X ? 

not all options are - 

also, they never really change, what's the value of making them configurable ?